web_client/src/components/ControlPanel.vue

```diff
@@ -206,52 +206,52 @@ export default {
             elevation="3"
           >
             <div class="d-flex">
-              <v-container>
-                <v-row no-gutters>
-                  <v-col cols="3">
+              <div
+                class="d-flex px-5 py-3"
+                style="flex-direction: column; flex-grow: 3"
+              >
+                <div class="current-info-container">
+                  <div>
                     Project:
-                  </v-col>
-                  <v-col cols="9">
+                  </div>
+                  <div id="projectName">
```

Comment:
Using `id` in a component is probably a code smell -- they are supposed to be globally unique in the DOM, so it makes this component essentially a singleton. I suggest we use a class instead.

Reply:
I see, okay. I thought it would make the XPath selection more granular, but I understand your point. I'll switch it back to a class selection

Reply:
Changed in [be0b4a9](https://github.com/OpenImaging/miqa/pull/563/commits/be0b4a95cff91f9a596ed7261c40aa1e336f23b1)